frontend/components/footer.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Footer = ({ baseUri }) => (
+  <section>
+    <h2 id="like-this">Like This?</h2>
+
+    <p>
+      Tell your favorite badge service to use it! <br />
+      And tell us, we might be able to bring it to you anyway!
+    </p>
+    <p>
+      <object
```

Comment:
Why `<object>` rather than `<img>`?

Reply:
It makes embedded links work! Probably that belongs in a Shields FAQ somewhere…